spec/lib/gistance/request_spec.rb

```diff
@@ -23,13 +23,28 @@
 
       assert_requested :get, 'https://maps.googleapis.com/maps/api/distancematrix/json?'\
         'language=en&'\
-        'unit=metric&'\
+        'units=metric&'\
         'sensor=false&'\
         'client=gme-fuubar&'\
         'channel=test&'\
-        'signature=wA6neJ6IkHTs6BxFt6ves7hxsZI%3D'
+        'signature=JlBk9pESGAhVTtCZbidG8Ss2fbM%3D'
 
       VCR.turn_on!
     end
+
+    it 'respects overridden options' do
```

Comment:
I added this spec because I initially thought the overridden unit type wasn't being used. Even though it was being used just not being acknowledged by Google because of the incorrect param (`unit` instead of `units`), I kept the spec to provide an assertion that overridden options are indeed used.